src/py/flwr/common/message.py

```diff
@@ -189,6 +221,42 @@ def content(self, value: RecordSet) -> None:
         """Set content."""
         self._content = value
 
+    @property
+    def error(self) -> Error:
+        """Error captured by this message."""
+        return self._error
+
+    @error.setter
+    def error(self, value: Error) -> None:
+        """Set error."""
+        self._error = value
+
+    def construct_error_message(
```

Comment:
If we want to keep the `create_error_message`, then we should rename `create_reply` to `create_reply_message`:
```suggestion
    def create_error(
```